src/components/Chart/ContextBox.js

```diff
@@ -0,0 +1,126 @@
+// Copyright 2017 Lobbywatch
+// BSD-3-Clause
+// https://github.com/lobbywatch/website/blob/master/src/components/ContextBox.js
+
+import React from 'react'
+import {css} from 'glamor'
+import PropTypes from 'prop-types'
+
+import colors from '../../theme/colors'
+import { Label } from '../Typography'
+import { sansSerifRegular14 } from '../Typography/styles'
+
+const boxStyle = css({
```

Comment:
Diverges a bit from the visual design of Republik:

- `boxShadow` is too fuzzy
- `borderRadius` is unusual